modules/compliance-tailored-profiles.adoc

```diff
@@ -65,6 +65,9 @@ spec:
 |`disableRules`
 |A list of name and rationale pairs. Each name refers to a name of a rule object that is to be disabled. The rationale value is human-readable text describing why the rule is disabled.
 
+|`manualRules`
+| A list of name and rationale pairs. Each name refers to a name of a rule object that is to be disabled. The rationale value is human-readable text describing why the rule is disabled. When a rule is added as a manual rule, it will always show manual as the check result status, and remediation will not be generated.
```

Comment:
A rule in this list wouldn't be disabled (that's what the `disableRules` attribute is for). Instead, those rule would always return MANUAL no matter what.

I know the upstream documentation is worded in a confusing manner, feel free to send a PR fixing the doc/crds.md, it would be appreciated, but the upstream docs say "Each name refers to a name  of a `Rule` object that is supposed to be disabled for its automated check", the "for its automated check" part is trying to say that the rule would always return MANUAL.

As per an example, this might be a good one:
```
apiVersion: compliance.openshift.io/v1alpha1
kind: TailoredProfile
metadata:
  name: rhcos4-manual-scc-check
spec:
  extends: ocp4-cis
  description: This profile extends ocp4-cis by forcing the SCC check to always return MANUAL
  title: OCP4 CIS profile with manual SCC check
  manualRules:
    - name: ocp4-cis-scc-limit-container-allowed-capabilities
      rationale: We use third party software that installs its own SCC with extra privileges
```

Reply:
Thanks for your help @jhrozek! 

I have rewarded the `manualRules` description and added the example. Please let me know if this is correct or if changes need to be made. 

As far as the  doc/crds.md, I will certainty look into getting it fixed and submitting a separate PR. :)